Assets/SC_Cube.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SC_Cube : MonoBehaviour
+{
+    private Transform transform;
+
+    public float rotationspeed;
+    // Start is called before the first frame update
+    void Start()
+    {
+        transform = GetComponent<Transform>();
+    }
```

Comment:
There is no need to initialize the Transform using the Start() method and the GetComponent<Transform>() method. Instead, it will be possible to get a Transform object anywhere in the class using gameObject.transform